lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,17 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $user = user_load($data->uid);
+    $this->user = [
+      'drupal_id' => $data->uid,
```

Comment:
We've been using `drupal_id` for this in Northstar, and I think it'd be a bit of an ordeal to change now. I'd definitely been trying to preference using "Phoenix" whenever possible otherwise though.